guided-trials-default-custom-user-data/default-custom-user-data.gradle

```diff
@@ -135,6 +135,17 @@ void addCiMetadata() {
             buildScan.value 'CI agent ID', agentId
         }
     }
+
+    if (isGitHubActions()) {
+        def repo = System.getenv('GITHUB_REPOSITORY')
+        def runId = System.getenv('GITHUB_RUN_ID')
+        if (repo && runId) {
+            buildScan.link 'CI job', "https://github.com/$repo/actions/runs/$runId"
+        }
+        if (System.getenv('GITHUB_HEAD_REF')) {
+            buildScan.tag 'Fork'
+        }
+    }
```

Comment:
can we change the things to be more like we do for e.g. Jenkins ?

buildScan.link 'GitHub actions build', "https://github.com/$repo/actions/runs/$runId"

buildScan.value 'CI build number', System.getenv('GITHUB_RUN_NUMBER')

// do sth with GITHUB_ACTION
def ghActionLabel = 'GitHub action'
def ghActionName = System.getenv('GITHUB_ACTION')
buildScan.value ghActionLabel, ghActionName
addCustomValueSearchLink 'GitHub action build scans', [(ghActionLabel): ghActionName]

// Do sth with the GITHUB_WORKFLOW
def ghActionWorkflowLabel = 'GitHub workflow'
def ghActionWorkflowName = System.getenv('GITHUB_WORKFLOW')
buildScan.value ghActionWorkflowLabel, ghActionWorkflowName
addCustomValueSearchLink 'GitHub action build scans', [(ghActionWorkflowLabel): ghActionWorkflowName]



=> not totally familiar with GH actions, but would those be useful ?

Reply:
I meant to do this but misunderstood what `CI job` does for Bamboo and CircleCI.

I think it should be "GitHub actions run" for the first link. `GITHUB_ACTION` is just a number, I don't think that's very useful, but `GITHUB_WORKFLOW` might. I'll change it accordingly.

Reply:
yeah, hard to check without using GH actions
but if you checked on a real usage, then I'm ok